frontend/lib/src/theme/getColors.ts

```diff
@@ -98,6 +103,9 @@ export const createEmotionColors = (genericColors: {
     dataframeHeaderBackgroundColor: derivedColors.bgMix,
 
     headingColor: genericColors.bodyText,
+
+    // @ts-expect-error -- defaultCategoricalColors is a string[] vs. string
+    chartCategoricalColors: defaultCategoricalColors,
```

Comment:
That's a bit annoying :( But probably will need a refactoring around how we type `theme.colors`, which is outside of the scope of this PR. I added a tech debt item [here](https://www.notion.so/snowflake-corp/Use-proper-typing-for-theme-colors-instead-of-string-map-21f7170bb41680ff957ffb166c25e13b).